contracts/btc-finality/src/multitest/suite.rs

```diff
@@ -339,7 +356,7 @@ impl Suite {
         pubrand_signature: &[u8],
     ) -> anyhow::Result<AppResponse> {
         self.app.execute_contract(
-            Addr::unchecked("anyone"),
+            Addr::unchecked(USER_ADDR),
```

Comment:
Nit: This can be literally anyone. Setting a specific address there is misleading.

Reply:
Added a docstring to this variable so that it's clear